content/en/case-studies/cricket-analytics.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Case Study: Cricket Analytics, the game changer!"
+sidebar: false
+---
+
+{{< figure src="/images/content_images/cs/ipl-stadium.png"
+           caption="**IPLT20, the biggest Cricket Festival in India**"
+           alt="Indian Premier League Cricket cup and stadium"
+           attr="*(Image credits: IPLT20 (cup and logo) & Akash Yadav (stadium))*"
+           attrlink="https://unsplash.com/@aksh1802" >}}
+
+<blockquote cite="https://www.scoopwhoop.com/sports/ms-dhoni/">
+    <p>You don't play for the crowd, you play for the country.</p>
+    <footer align="right">—M S Dhoni, <cite>International Cricket Player, ex-captain, Indian Team, plays for Chennai Super Kings in IPL</cite></footer>
+</blockquote>
```

Comment:
It could be nice to replace this with a quote related to NumPy. I think of all the links in this case study, the _Stats LLC_ one looks most substantial. @shaloo have you tried contacting http://patricklucey.com/index.html ?

Reply:
Initiated email conversation regarding the same.

Reply:
Waiting on email response. We can update it once I get more insights from Patrick later as a new / fresh PR.  May be we can close this one and open a new issue once I hear from Patrick.

Reply:
Ralf, can we merge cricket case study updates now?  I haven't heard back from most of those who I contacted regarding this, Patrick included.  Once it shows up on the site, there will be more takers to provide feedback or better conten, I think.